contracts/PangolinExchange.sol

```diff
@@ -26,16 +27,15 @@ contract PangolinExchange is Ownable, IAssetsExchange {
 
   /* ========= MODIFIERS ========= */
 
-  modifier RefundRemainder {
-    _;
+  function refundAvaxBalance() private {
```

Comment:
`nonReentrant`?

Reply:
There is no point for re-entrancy protection in private functions